package.json

```diff
@@ -4784,6 +4785,15 @@
           "csharp"
         ],
         "aiKey": "0c6ae279ed8443289764825290e4f9e2-1a736e7c-1324-4338-be46-fc2a58ae4d14-7255"
+      },
+      {
+        "type": "remotecoreclr",
```

Comment:
Suggestion: I think people might find this term and description confusing since there is already a different remote debugging story for 'coreclr'.  How about "coreclr_mobile" or something like that?